cli/cli.go

```diff
@@ -596,6 +599,23 @@ func Priam(args []string, defaultCfgFile string, infoW, errorW io.Writer) {
 						return nil
 					},
 				},
+				{
+					Name: "aws", Usage: "Use ID token to update credentials in the AWS CLI configuration file", ArgsUsage: "<aws-role-name>",
```

Comment:
We should be clear about the role: looks like you already prepend  "arn:aws:iam::044114111530:role/"